backend/api/Services/Models/IsarMediaConfig.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json.Serialization;
+namespace Api.Services.Models
+{
+    public struct IsarMediaConfig
```

Comment:
If this object is to pass between the frontend and backend over signalr. It should not be named with the Isar prefix

Reply:
Good point. Renamed.